Jenkinsfile

```diff
@@ -450,10 +449,10 @@ def buildPackages() {
   def stashPaths = [
       'bazel-bin/src/tools/auth_token_gen',
       'bazel-bin/test/grpc/grpc-test-client',
-      'bazel-bin/test/grpc/interop-client',
-      'bazel-bin/external/org_golang_google_grpc/stress/metrics_client/metrics_client',
-      'bazel-bin/external/org_golang_google_grpc/interop/server/server',
-      'bazel-bin/external/org_golang_google_grpc/stress/client/client',
+      'bazel-bin/test/grpc/linux_amd64_stripped/interop-client',
```

Comment:
if --config=asan or debug,the file name will without linux_amd64_stripped.   do we need to stash or save them?

Reply:
Jenkins stores Release version binary to GCS bucket for e2e testing, so we do not neet stash asan or debug one.